Frameworks/PyTorch/Getting-Started-Samples/PyTorch_HelloWorld/sample.json

```diff
@@ -0,0 +1,22 @@
+{
+  "guid": "3E6FD81F-76A0-430C-AD82-0F94A45F93DC",
+  "name": "PyTorch Hello World",
+  "categories": ["Toolkit/Intel® AI Analytics Toolkit/PyTorch"],
+  "description": "This sample shows how to train a PyTorch model and run inference with oneMKL and oneDNN.",
+  "builder": ["cli"],
+  "languages": [{"python":{}}],
+  "os":["linux"],
+  "ciTests": {
+	"linux": [
+	{
+		"env": ["source /opt/intel/oneapi/setvars.sh --force" ],
```

Comment:
Why would you need to re-source `setvars.sh`?

Reply:
we want to make sure the environment is setting. thx

Reply:
This is not required. `setvars.sh` is sourced in CI before executing sample tests. Please, remove this line here and for the other sample. Thanks.

Reply:
@yinghu5 @louie-tsai can you please address the setvars.sh issue described by Mkitez

Reply:
taken care of